modules/ascat/main.nf

```diff
@@ -64,8 +66,8 @@ process ASCAT {
     ascat.prepareHTS(
         tumourseqfile = "$input_tumor",
         normalseqfile = "$input_normal",
-        tumourname = "Tumour",
-        normalname = "Normal",
+        tumourname = paste0("$prefix", "_tumour"),
+        normalname = paste0("$prefix", "_normal"),
```

Comment:
```suggestion
        tumourname = paste0("$prefix", ".tumour"),
        normalname = paste0("$prefix", ".normal"),
```

Reply:
Sure I can change the underscores, but this won't work for the `alleleFrequency, `logR`and `BAF` files as the underscores are prepended internally in ASCAT.